src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -33,6 +35,12 @@ AbstractTableGenerator::AbstractTableGenerator(const std::shared_ptr<BenchmarkCo
 void AbstractTableGenerator::generate_and_store() {
   Timer timer;
 
+  // Encoding table data and generating table statistics are time consuming processes. To reduce
```

Comment:
State that we store current scheduler for the case it’s the single threaded one. Otherwise this change of the scheduler looks confusing to others.

Reply:
Narf, sorry, the information was already there but was too stupid to read mobile GitHub. Nonetheless, the now added comment helps.